src/PerformanceTests/Common/EnvironmentStats.cs

```diff
@@ -31,5 +32,11 @@ public static void Write()
         log.InfoFormat("EnableDnsRoundRobin: {0}", ServicePointManager.EnableDnsRoundRobin);
         log.InfoFormat("Expect100Continue: {0}", ServicePointManager.Expect100Continue);
         log.InfoFormat("UseNagleAlgorithm: {0}", ServicePointManager.UseNagleAlgorithm);
+
+        log = LogManager.GetLogger("ENVVAR");
+        foreach (var item in Environment.GetEnvironmentVariables().OfType<DictionaryEntry>().OrderBy(x=>x.Key))
+        {
+            log.DebugFormat("{0}={1}", item.Key, item.Value);
```

Comment:
👎 this is a security risk

Reply:
That is why it is only done with log level DEBUG, as mentioned in the commit message.

Reply:
is this really necessary?

Reply:
I'd vote to remove it. +1 on removing the logging of the connection string as well, although that's not the same level of risks, as it only exposes a specific, selected env var and not all other, potential personal env vars on the users machine.